.gitignore

```diff
@@ -161,14 +161,9 @@ node_modules
 /package-lock.json
 
 # Generated outputs
-dist/data/
-dist/output/
-src/data/
-src/output/
+data/
+output/
 
 # Debugging files
 debugme_script.sh
-debugme.sh
-
-
-
+debugme.sh
```

Comment:
```suggestion
debugme.sh

```

Reply:
Solved in commit: [83444f5](https://github.com/AbsaOSS/living_doc_generator/commit/83444f5c51113dce848058751f4bf50f28d8c757)